### PR TITLE
TimeSerie option ( Min, Max, Decimals )

### PR DESCRIPTION
**Pull request by vcandeau**
## What does this do?
[ADD] Minimun options for Timeseries
[ADD] Maximun options for Timeseries
[ADD] Number of decimals displays options for Timeseries

## Why is it a good idea?
Give ability to adjust the watch size of the grid

## Context
Answering the issue [474](https://github.com/weaveworks/grafanalib/issues/474)

### CODE REVIEW

File: grafanalib/core.py [hunk 1]
```diff
@@ -1575,6 +1575,9 @@ class TimeSeries(Panel):
         single (Default), multi, none
     :param unit: units
     :param thresholdsStyleMode: thresholds style mode off (Default), area, line, line+area
+    :param valueMin: Minimun value for Panel
+    :param valueMax: Minimun value for Panel
```
Comment (JamesGibo) [inline grafanalib/core.py:1579]:
```suggestion
    :param valueMax: Maximum value for Panel
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -6,7 +6,9 @@ x.x.x (TBC)
 ===========
 
 * Added ...
-
+* Added Minimun option for Timeserie
+* Added Maximun option for Timeserie
+* Added Number of decimals displays option for Timeserie
```
Comment (JamesGibo) [inline CHANGELOG.rst:11]:
```suggestion
* Added Minimum option for Timeseries
* Added Maximum option for Timeseries
* Added Number of decimals displays option for Timeseries
```

**Review by JamesGibo — CHANGES_REQUESTED** [review]

**Comment by vcandeau** [comment]
Done regardless to your suggestion @JamesGibo

File: grafanalib/core.py [hunk 3]
```diff
@@ -1574,6 +1574,9 @@ class TimeSeries(Panel):
         single (Default), multi, none
     :param unit: units
     :param thresholdsStyleMode: thresholds style mode off (Default), area, line, line+area
+    :param valueMin: Minimum value for Panel
+    :param valueMax: Minimum value for Panel
```
Comment (JamesGibo) [inline grafanalib/core.py:1578]:
```suggestion
    :param valueMax: Maximum value for Panel
```

**Comment by vcandeau** [comment]
Fix

**Comment by cristifalcas** [comment]
Any traction for this? We also hit this issue

**Comment by cristifalcas** [comment]
ping @vcandeau :)

**Comment by vcandeau** [comment]
No response on the fix after reviewing

**Comment by vcandeau** [comment]
For me it's done and working just need to be merge

**Comment by be-hase** [comment]
@JamesGibo 
could you check this PR? I want to use min&max option.

**Comment by fzyzcjy** [comment]
Is there any updates? Thanks

**Review by JamesGibo — APPROVED** [review]
LGTM

**Pull request merged**